lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -776,6 +796,103 @@
             }
           }
 
+          .xseries-action{
+            .xseries-msg{
+              @include float(left);
+              width: flex-grid(9, 12);
+            }
+
+            p+p{
+              margin-top: 0;
+            }
+
+            .message-copy{
+              font-weight: 500;
+            }
+
+            .message-copy-bold{
+              font-weight: 600;
+            }
+
+            .xseries-border-btn {
+              @extend %btn-pl-black-border;
+              @include float(right);
+              position: relative;
+              left: 10px;
+              padding: ($baseline*0.4) ($baseline*0.6);
+              background-image: none ;
+              text-shadow: none;
+              box-shadow: none;
+              text-transform: none;
+
+              .action-xseries-icon{
+                @include float(left);
+                display: inline;
+
+                @include margin-right($baseline*0.4);
+                background: url('#{$static-path}/images/icon-sm-xseries-black.png') no-repeat;
+                background-color: transparent;
+
+                width: ($baseline*1.1);
+                height: ($baseline*1.1);
+              }
+              &:hover,
+              &:focus {
+
+                .action-xseries-icon{
+                  @include float(left);
+                  display: inline;
+
+                  @include margin-right($baseline*0.4);
+                  background: url('#{$static-path}/images/icon-sm-xseries-white.png') no-repeat;
+                  background-color: transparent;
+
+                  width: ($baseline*1.1);
+                  height: ($baseline*1.1);
+                }
+              }
+            }
+
+            .xseries-base-btn {
+              @extend %btn-pl-black-base;
```

Comment:
+1 This kind of extension is what I was asking for / about for the button above. That being said, it does seem like a lot of overrides to accomplish the button styling goal. Is there a @extend %btn-pl-black-secondary ? That would be the button styling we'd like to echo instead of the -base variant.